lib/src/model/container.dart

```diff
@@ -153,8 +153,16 @@ abstract class Container extends ModelElement
   late final List<Field> publicConstantFieldsSorted =
       publicConstantFields.toList(growable: false)..sort(byName);
 
+  /// The total list of public enum values.
+  ///
+  /// This is defined on [Container] instead of just [Enum], because the
+  /// `ContainerSidebar` Mustache template needs to refer to this field.
```

Comment:
I think this is just the side effect of using Mustache templates, but this doesn't feel great. I wouldn't choose to have `publicEnumValues` on the superclass of enum. 

But alas.

Reply:
Oh yeah. A bummer. There are a few of these. More than being an issue with Mustache templates, it is an issue specifically with our implementation. I haven't looked at this stuff in years, and it might be ripe for fixing. :D

Reply:
I looked a bit and remembered. A bit of history 😁 

When I initially wrote Mustachio, our mirrorless Mustache renderer, I envisioned it might be extracted into its own package some day. But I wanted to bake it in dartdoc first. That was a good decision because a few of the little designs are optimized for dartdoc. We have never opened it as a stand-alone Mustache renderer, and that's fine. One less headache 😬 .

It also kind of opens some doors. We can add some custom syntax on Mustache. There are two I've wanted:

* A boolean block based on static typing. `{{ #foo is Foo }}`. There are a handful of places where it would come in handy, and we wouldn't need as much shoe-horning like this.
* An awaited Future value. `{{ await foo }}` maybe. In particular, one reason we have to pre-process all doc comments is so that they are waiting patiently on each ModelElement by the time Mustachio comes by to read them. Mustachio cannot currently ask an object for a value which is Future-typed.